feature-libs/cart/base/core/store/effects/multi-cart.effect.ts

```diff
@@ -58,8 +58,22 @@ export class MultiCartEffects {
           }
           break;
         }
-        case CartActions.LOAD_CART_SUCCESS:
-        // point to `temp-${uuid}` when we are creating/merging cart
+        case CartActions.LOAD_CART_SUCCESS: {
+          if (action?.payload?.extraData?.active) {
```

Comment:
To reviewer: In here, we make sure the active cart is not saved cart, in case the cart's `saveTime` is set in backend.